### PR TITLE
File upload

### PR DESCRIPTION
**Pull request by rpallavisharma**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

The file upload example is a common scenario and an example of it is added . 
### Motivation and Context

The selenium documentation was missing this commonly used scenario. 
### Types of changes

- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [x] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  {{< /tab >}}
+  {{< tab header="JavaScript" >}}
+// This feature is not implemented - Help us by sending a pr to implement this feature
+  {{< /tab >}}
+  {{< tab header="Kotlin" >}}
+  // This feature is not implemented - Help us by sending a pr to implement this feature
+{{< /tab >}}
+{{< /tabpane >}}
+
+Thus the above example code helps to understand how we can upload a file using selenium.
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md]: Selenium

File: website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                driver.Quit();
+
+            }
+
+            }
+}
+
+  {{< /tab >}}
+  {{< tab header="Ruby" >}}
+// This feature is not implemented - Help us by sending a pr to implement this feature
```
Comment (harsha509) [inline website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md]: Maybe 

Help us by sending a pr for code sample

feature is available, what we're looking for is code sample I assume!

**Review by harsha509 — CHANGES_REQUESTED** [review]

File: website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md (hunk 2 above)
Comment (rpallavisharma) [reply website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md]: you are right Harsha. Modified it.

File: website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md (hunk 1 above)
Comment (harsha509) [reply website_and_docs/content/documentation/webdriver/additional_features/fileupload.zh-cn.md]: Hi @pallavigitwork , can you make this change too !

**Review by harsha509 — APPROVED** [review]

**Comment by harsha509** [comment]
Thank you @pallavigitwork!

**Comment by rpallavisharma** [comment]
Thanks for the help and suggestions @harsha509

**Pull request merged**